cmd/opentelemetry/app/exporter/elasticsearchexporter/esmodeltranslator/modeltranslator.go

```diff
@@ -222,21 +222,15 @@ func references(links pdata.SpanLinkSlice, parentSpanID pdata.SpanID, traceID db
 }
 
 func convertSpanID(spanID pdata.SpanID) (dbmodel.SpanID, error) {
-	spanIDInt, err := tracetranslator.BytesToUInt64SpanID(spanID.Bytes())
-	if err != nil {
-		return "", err
-	}
+	spanIDInt := tracetranslator.BytesToUInt64SpanID(spanID.Bytes())
 	if spanIDInt == 0 {
 		return "", errZeroSpanID
 	}
 	return dbmodel.SpanID(fmt.Sprintf("%016x", spanIDInt)), nil
```

Comment:
This probably would be more efficient:
```go
import "encoding/hex"

bytes := spanID.Bytes()
dst := make([]byte, hex.EncodedLen(len(src)))
hex.Encode(dst, src)
return dbmodel.SpanID(dst)
```

Reply:
According to benchmark run on my device, this change gave 70ns vs previous 170ns